include/cocaine/format.hpp

```diff
@@ -43,11 +43,11 @@ substitute(boost::format&& message, const T& argument, const Args&... args) {
 template<class... Args>
 static inline
 std::string
-format(const std::string& format, const Args&... args) {
+format(const std::string& format, Args&&... args) {
```

Comment:
Strictly speaking there is no need for universal references here. Moreover it can hurt.

Given for example you have a large struct with overloaded `operator<<` - `foo_t`, and `sizeof(foo_t) == 1000`. Then when we type `format(foo_t())` the argument is deduced as an rvalue.

Thus on every `forward` we "move" those 1000 bytes by copying, which is worse than just passing a const reference.